pkg/resolver/rbacHelper.go

```diff
@@ -83,3 +83,11 @@ func matchManagedCluster(managedClusters []string) exp.BooleanExpression {
 	//managed clusters
 	return goqu.C("cluster").Eq(goqu.Any(pq.Array(managedClusters)))
 }
+
+func getKeysStructMap(structArrayMap map[string]struct{}) []string {
```

Comment:
This function is removed in the next PR https://github.com/stolostron/search-v2-api/pull/88. It is incorporated into the getKeys() function.